packages/braid-design-system/src/lib/components/Inline/Inline.screenshots.tsx

```diff
@@ -342,5 +342,40 @@ export const screenshots: ComponentScreenshot = {
         </Inline>
       ),
     },
+    {
+      label:
+        'Test - truncation should work an (all examples should truncate below)',
```

Comment:
Its a pretty bad label, lets try again: 
```suggestion
        'Test - truncation should be visible on all examples below',
```